clients/unshaded/src/main/java/tachyon/client/tcp/TCPRemoteBlockReader.java

```diff
@@ -66,8 +67,9 @@ public ByteBuffer readRemoteBlock(String host, int port, long blockId, long offs
       }
       LOG.info("Data " + blockId + " from remote machine " + address + " received");
 
-      if (recvMsg.getBlockId() < 0) {
-        LOG.info("Data " + recvMsg.getBlockId() + " is not in remote machine.");
+      if (recvMsg.getStatus() != RPCResponse.Status.SUCCESS) {
+        LOG.error("Error in response for blockId: " + recvMsg.getBlockId() + " message: "
```

Comment:
Is warn more appropriate than error here? Since it ends up handling the case without user intervention?